Doc/library/getopt.rst

```diff
@@ -7,18 +7,22 @@
 
 **Source code:** :source:`Lib/getopt.py`
 
+.. deprecated:: 3.13
+   The :mod:`getopt` module is deprecated and will not be developed further;
+   development will continue with the :mod:`argparse` module.
+
 .. note::
 
    The :mod:`getopt` module is a parser for command line options whose API is
-   designed to be familiar to users of the C :c:func:`getopt` function. Users who
-   are unfamiliar with the C :c:func:`getopt` function or who would like to write
+   designed to be familiar to users of the C :c:func:`!getopt` function. Users who
+   are unfamiliar with the C :c:func:`!getopt` function or who would like to write
    less code and get better help and error messages should consider using the
    :mod:`argparse` module instead.
```

Comment:
```suggestion
   designed to be familiar to users of the C :c:func:`!getopt` function.
   Using the :mod:`argparse` module is encouraged instead,
   as it offers better help and error messages while writing less code.
```

Tweaked this a bit to account to nudge users more toward `argparse`, given the deprecation, rather than just naming specific categories of users that should "consider using" it instead, as well as be more concise and less duplicative/repetitive.

Reply:
Note that I still think these changes (or perhaps a revised version) would be a good idea if we decide not to actually deprecate it, to make this clearer, more concise and a bit more of a nudge.